### PR TITLE
fix tray menu translation

### LINKED ISSUE #211: Translations doesn't apply to context menu in notification area
Context menu in notification area contains 3 string (Open, Settings, Exit) in English only.

### PR DESCRIPTION
**Pull request by taooceros**
fix #211

### CODE REVIEW

File: Flow.Launcher/MainWindow.xaml.cs [hunk 1]
```diff
@@ -53,11 +53,12 @@ private void OnClosing(object sender, CancelEventArgs e)
         private void OnInitialized(object sender, EventArgs e)
         {
             // show notify icon when flowlauncher is hided
```
Comment (jjw24) [inline Flow.Launcher/MainWindow.xaml.cs:55]: this comment goes with the method right?

Comment (jjw24) [reply Flow.Launcher/MainWindow.xaml.cs:55]: also while you are at it please change word from 'hided' to 'hidden' please

**Comment by jjw24** [comment]
putting it there requires a restart, is it possible to put it somewhere where restart is not required like the settings text?

File: Flow.Launcher/MainWindow.xaml.cs (hunk 1 above)
Comment (taooceros) [reply Flow.Launcher/MainWindow.xaml.cs:55]: Sure.

**Comment by taooceros** [comment]
> putting it there requires a restart, is it possible to put it somewhere where restart is not required like the settings text?

Not sure what you mean......

**Comment by jjw24** [comment]
The InitializeNotificationIcon call, that will only get called when program is started. So when you change the language, the translation on the tray menu will not happen untill next start. 

Am wondering if there is a way to have it so when language is changed that method is called so when user changes the language the tray menu changes straight away as well

**Comment by taooceros** [comment]
Understand, maybe we can create a public method used to update tray text when language has changed. I will take a look on that later.

**Comment by taooceros** [comment]
> The InitializeNotificationIcon call, that will only get called when program is started. So when you change the language, the translation on the tray menu will not happen untill next start.

> Am wondering if there is a way to have it so when language is changed that method is called so when user changes the language the tray menu changes straight away as we

Done it

**Comment by jjw24** [comment]
👍

**Review by jjw24 — APPROVED** [review]

**Pull request merged**